areader/reader.go

```diff
@@ -656,8 +656,12 @@ func (ar *Reader) setInstallers(upd []artifact.UpdateType, augmented bool) error
 				ar.installers[i] = w.NewInstance()
 			}
 		} else if ar.ForbidUnknownHandlers {
-			return fmt.Errorf("Cannot load handler for unknown Payload type '%s'",
-				update.Type)
+			errstr := fmt.Sprintf("Artifact Payload type '%s' is not supported by this Mender Client", update.Type)
```

Comment:
@oleorhagen: You never submitted this to mender-artifact, only into the vendor part of mender.

Reply:
Oh lord, I'm sorry!

Reply:
It left me confused for a while! :laughing: